test/domain_bridge/test_qos_matching.cpp

```diff
@@ -38,49 +40,67 @@ class TestDomainBridgeQosMatching : public ::testing::Test
 constexpr std::size_t TestDomainBridgeQosMatching::domain_1_;
 constexpr std::size_t TestDomainBridgeQosMatching::domain_2_;
 
+std::ostream& operator<<(std::ostream& os, const rclcpp::TopicEndpointInfo& t)
+{
+  os << t.node_namespace() << '/' << t.node_name() << ": " << t.topic_type();
+  return os;
+}
+
```

Comment:
This is some code I'm using for debugging the tests. Will remove in a later commit.